packages/evm-client-viem/src/contract/createReadWriteContract.ts

```diff
@@ -8,12 +8,16 @@ import {
   CreateReadContractOptions,
 } from 'src/contract/createReadContract';
 import { createSimulateContractParameters } from 'src/contract/utils/createSimulateContractParameters';
-import { Abi, WalletClient } from 'viem';
+import { Abi, Address, WalletClient } from 'viem';
 
 export interface ReadWriteContractOptions<TAbi extends Abi = Abi>
   extends CreateReadContractOptions<TAbi> {
   walletClient: WalletClient;
   readContract?: ReadContract<TAbi>;
+  /**
+   * Turn this off for debugging
+   */
+  validateBeforeWrite?: boolean;
```

Comment:
nit: Use exact name to make connection clearer.

```suggestion
  simulateBeforeWrite?: boolean;
```